util/gitutil/detectgit.go

```diff
@@ -302,6 +302,17 @@ func detectGitAuthor(ctx context.Context, dir string) (string, error) {
 	return strings.SplitN(outStr, "\n", 2)[0], nil
 }
 
+// GlobalEmail returns the user's currently configured (global) email address
+func GlobalEmail(ctx context.Context) (string, error) {
```

Comment:
This is wrong now right? It's not really the global email

Reply:
Good point. Thanks.